src/System.ComponentModel.Composition/ref/Compat/System.ComponentModel.Composition.csproj

```diff
@@ -7,24 +7,17 @@
     <PackageTargetFramework>net46</PackageTargetFramework>
     <NuGetTargetMoniker>.NETFramework,Version=v4.6</NuGetTargetMoniker>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ReferenceAssemblyOutputPath Condition="'$(ReferenceAssemblyOutputPath)' == ''">$(BaseOutputPath)ref/</ReferenceAssemblyOutputPath>
+    <AssemblyName Condition="'$(AssemblyName)' == ''">$(MSBuildProjectName)</AssemblyName>
+    <OutputPath>$(ReferenceAssemblyOutputPath)$(AssemblyName)/$(AssemblyVersion)/Compat/</OutputPath>
+    <IntermediateOutputPath>$(BaseIntermediateOutputPath)ref/$(AssemblyName)/$(AssemblyVersion)/Compat/</IntermediateOutputPath>
```

Comment:
yes, we need to set both in some cases because in projects like System.IO.Compression there is already a file being dropped at `$(BaseIntermediateOutputPath)ref/$(AssemblyName)/$(AssemblyVersion)` so that would cause binclashes errors. Because of this, I thought that it would be better to be consistent and have all of the Contracts behave this way.